src/components/slider/handle.js

```diff
@@ -61,7 +49,13 @@ class Handle extends Component {
             onMouseEnter: this.onMouseEnter,
             onMouseLeave: this.onMouseLeave,
           })}
-        />
+        >
+          {(mouseOver || isActive) && !disabled &&
+            <div className="tooltip-con" style={{left: 0}}>
```

Comment:
Those this affect the position ?
This tootlip should be centered and had some padding around it.

Can you post a screen preview ?

Reply:
Yes, I aligned it with the rest of tooltips, the `left: 0` is needed to avoid horizontal displacement.
![image](https://user-images.githubusercontent.com/7670276/67220588-572ecf00-f42a-11e9-95a8-589cdc634c21.png)

Reply:
I guess is ok for now, is there anyway it can be truly aligned to center ?

Reply:
Let me some minutes, I don't like either, but I didn't want to change the tooltip settings too much.

Reply:
Yeah I think the current styles aren't doing a great job: https://github.com/btzr-io/Villain/issues/94

They require a full refactoring :stuck_out_tongue:

Reply:
Now is like this
![image](https://user-images.githubusercontent.com/7670276/67222818-61eb6300-f42e-11e9-9619-98270db8717a.png)

Reply:
Looks better but the problem is the dynamic text, I think it will break again once it reach a number like 10 or 100...

Reply:
Anyway I dont  think there is nothing we can do about it on pure `css` will probably require some `js`